touchforms/formplayer/api.py

```diff
@@ -318,7 +318,7 @@ def sync_db(username, domain=None, auth=None):
     data = {
```

Comment:
any reason why we can't make the function signature here `sync_db(username, domain, auth)`? it seems like it always gets called with those and it shouldn't be optional